examples/main.rs

```diff
@@ -5,45 +5,55 @@ use microwfc::*;
 use rand::thread_rng;
 
 #[derive(Clone, PartialEq, Eq, Debug)]
-enum Test {
-    Stone,
+enum Tile {
+    Water,
     Dirt,
     Grass,
 }
 
-impl PossibleValues for Test {
+impl Tile {
+    fn to_char(&self) -> char {
+        match self {
+            Tile::Dirt => '🟫',
+            Tile::Grass => '🟩',
+            Tile::Water => '🟦',
```

Comment:
Emojis might not be best here, maybe use an [ANSI color code](https://wikipedia.org/wiki/ANSI_escape_sequence#8-bit) and two spaces for a more uniform look?